src/tracing/trace_event.h

```diff
@@ -0,0 +1,1710 @@
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SRC_TRACING_TRACE_EVENT_H_
+#define SRC_TRACING_TRACE_EVENT_H_
+
+#include <stddef.h>
+
+#include "v8-platform.h"
```

Comment:
@mhdawson @joshgav At the diagnostics meeting today, we talked about potentially abstracting these operations on the V8 platform to a more general node platform. I do not feel strongly either way but I think it may be less churn to rewrite this once a node platform api is decided upon as opposed to starting it specifically for this use case. Either way, I wanted to continue the discussion where it could be recorded for those not at the summit.